src/zkevm_specs/util/arithmetic.py

```diff
@@ -1,10 +1,29 @@
 from __future__ import annotations
-from typing import Protocol, Sequence, Type, TypeVar, Union
-from functools import reduce
+from typing import Sequence, Protocol, Type, TypeVar, Union
 from py_ecc import bn128
 from py_ecc.utils import prime_field_inv
 
 
+def linear_combine(le_bytes: Sequence[Union[int, FQ]], base: FQ, range_check: bool = True) -> FQ:
+    """
+    Aggregate bytes into a single field element.
+    If we intend to use it as a commitment, the base must be a secured random number.
+    >>> r = 10
+    >>> assert linear_combine([1, 2, 3], r) == 1 * r**2 + 2 * r + 3
```

Comment:
Shouldn't this be `== 1 + r * 2 + r**2 + 3`?

Reply:
😱 nice catch

Reply:
Addressed in [a375590](https://github.com/privacy-scaling-explorations/zkevm-specs/pull/240/commits/a375590df25dd9318f4b34d676096adbde15e235)